pkg/api/proxy.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/dchest/siphash"
+	"github.com/gorilla/mux"
+
+	"zotregistry.dev/zot/pkg/api/config"
+)
+
+func ClusterProxy(config *config.Config) func(http.HandlerFunc) http.HandlerFunc {
+	return func(next http.HandlerFunc) http.HandlerFunc {
+		return http.HandlerFunc(func(response http.ResponseWriter, request *http.Request) {
+			requestPath := request.URL.Path
```

Comment:
I have no clue what happened to the formatting of this file. It looks just fine on the editor :)